src/routes/+page.svelte

```diff
@@ -1,3 +1,9 @@
+<script>
+    import Clock from "$lib/components/Clock.svelte";
+</script>
 <h1>Welcome to your library project</h1>
 <p>Create your package using @sveltejs/package and preview/showcase your work with SvelteKit</p>
 <p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
+<section>
```

Comment:
this will be removed later right?

Reply:
Ja vergeten. Ik ga het zo meteen veranderen.